vault/src/cancellation.rs

```diff
@@ -307,19 +311,6 @@ impl<P: IssuePallet + ReplacePallet + UtilFuncs + SecurityPallet + Clone> Cancel
 
         Ok(ret)
     }
-
-    /// Cached function to get the issue/replace period, in number of blocks until
-    /// it is allowed to be canceled
-    async fn get_cached_period<T: Canceller<P>>(&mut self) -> Result<u32, Error> {
```

Comment:
We need to refetch in case the period change, so we can't cache forever. The performance difference is anyway negligible because the period is only fetched after having fetched all open issues, which is more expensive.